package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "dashboard",
```

Comment:
Why was this removed?

Reply:
Updated ( it was removed due to my npm version being `7.5` )